frontend/src/lib/components/FullScreen.tsx

```diff
@@ -1,17 +1,17 @@
 import { useEffect } from 'react'
 
-export function FullScreen({ onExit }) {
+export function FullScreen({ onExit }: { onExit?: () => any }): null {
     const selector = 'aside.ant-layout-sider, .layout-top-content'
     useEffect(() => {
-        const myClasses = window.document.querySelectorAll(selector)
+        const myClasses = window.document.querySelectorAll(selector) as NodeListOf<HTMLElement>
 
         for (let i = 0; i < myClasses.length; i++) {
             myClasses[i].style.display = 'none'
         }
 
-        const handler = () => {
+        const handler = (): void => {
             if (window.document.fullscreenElement === null) {
-                onExit && onExit()
+                onExit?.()
```

Comment:
didn't know you could do this, cool!

Reply:
Yup, you can put this in a lot of places. `someArray?.[1]` was one that I didn't know of initially either.